tests/lang-elixir.yaml

```diff
@@ -0,0 +1,4 @@
+- desc: it should run elixir
+  command: [elixir,--version]
+  assert:
+  - status == 0
```

Comment:
Could you please change this test so that we can check the version of elixir?

Reply:
loading this morning and will test build